main.cpp

```diff
@@ -126,7 +129,22 @@ list<string> loadPlugins(list<string> plugins) {
                 ((void(*)()) sym)();
             }
         }
-    });
+    }
+
+    // Initialize our version string.
+    vector<string> versions = {SVERSION};
+    for (BedrockPlugin* plugin : *BedrockPlugin::g_registeredPluginList) {
+        // We need to call initialize to let the plugin set its version info.
+        plugin->initialize(args);
+        auto info = plugin->getInfo();
```

Comment:
I think I've used it in a couple other places to avoid writing out complex types like `const map<string, list<JSON::Value>>::iterator` or whatever.